src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -136,8 +136,8 @@ def _snapshot(self, fetcher: ResultFn, loader: ResultFn, *, force_refresh: bool)
                 cached_results = list(fetcher())
                 if cached_results:
                     return cached_results
-            except NotFound:
-                pass
+            except NotFound as e:
+                logger.debug(e.strerror)
```

Comment:
Typically we pass an error to a log as follows:

```suggestion
                logger.debug("Inventory table not found", exc_info=e)
```